### ISSUE TITLE
Quic web transport

### ISSUE THREAD

**Issue by gedw99** [alta/quic-go-mobile-example#2]
Hey @ydnar 

I saw your upstream PR to the golang QUIC repo . It’s still sitting there despite you bending over backwards.

i would like to get cracking on it even if it’s a bit crusty .

dobyou have a fork somewhere and what branch should I use if it ?

i want to take this swift demo and try it with GIO.  In gio it should just work. 

can you let me know please ? 

Also it would be cool to take a crack at web transport p2p. I have not found that in the main quic golang repo . Probably too bleeding edge still ..

**Comment by ydnar** [comment]
Thanks for writing. I think the best thing you could do is nudge the quic-go maintainers to express your interest, and I can follow up again.

Second, I suspect (hope?) that the next iOS and macOS rev will include API support for the datagram draft. It's in there (used for the private iCloud network), but not exposed via the public API.

**Comment by gedw99** [comment]
@ydnar 

I agree with the sentiment and will try to help encourage the quic golang team to move the PR into HEAD.

I think there is only one maintainer . It’s a pity golang quic is not getting a higher level of importance. 

Regarding iOS and Mac are you referring to web or native ( apps ) ?
